src/core/ext/filters/client_channel/lb_policy/xds/xds_override_host.cc

```diff
@@ -242,21 +243,18 @@ class XdsOverrideHostLb : public LoadBalancingPolicy {
     explicit SubchannelEntry(XdsHealthStatus eds_health_status)
         : eds_health_status_(eds_health_status) {}
 
-    ~SubchannelEntry() {
-      SubchannelWrapper* subchannel = GetSubchannel();
-      if (subchannel != nullptr) subchannel->Shutdown();
-    }
-
-    void SetSubchannel(RefCountedPtr<SubchannelWrapper> subchannel) {
+    void SetSubchannel(SubchannelWrapper* subchannel) {
+      ResetSubchannel();
```

Comment:
Now that we've changed the subchannel to shut down automatically when it loses all of its strong refs, I think we no longer need this line.

Reply:
Yep.